lib/Doctrine/Common/Collections/Collection.php

```diff
@@ -221,7 +221,7 @@ public function exists(Closure $p);
      *
      * @return Collection A collection with the results of the filter operation.
      *
-     * @psalm-param Closure(T=):bool $p
+     * @psalm-param Closure(T=, TKey):bool $p
```

Comment:
Hmm, optionality should be clarified here too

Reply:
Fixed.